core/conn/odb/src/odb.c

```diff
@@ -5491,9 +5491,11 @@ static void etabadd(char type, char *run, int id)
                     }
                 } else {                            /* not a load job */
                     etab[no].k = no;                /* record grandparent for copy/diff ops */
-                    if ( etab[no].ps ){
+                    if ( etab[no].ps > 1 ){
```

Comment:
Can you please explain why we don't need this IF block when etab[no].ps  =1 ? Is it it because for a extract/copy when we have one stream comparing rowset size to max number of rows is done elsewhere in this file?